x-pack/plugins/secops/public/apps/start_app.tsx

```diff
@@ -27,14 +28,16 @@ export const startApp = async (libs: AppFrontendLibs) => {
   const store = createStore();
 
   libs.framework.render(
-    <EuiErrorBoundary>
-      <ReduxStoreProvider store={store}>
-        <ApolloProvider client={libs.apolloClient}>
-          <ThemeProvider theme={{ eui: dark }}>
-            <PageRouter history={history} />
-          </ThemeProvider>
-        </ApolloProvider>
-      </ReduxStoreProvider>
-    </EuiErrorBoundary>
+    <I18nProvider>
+      <EuiErrorBoundary>
```

Comment:
I think EuiErrorBoundary should encapsulate I18nProvider, just in case something weird happens there. We can still show a message to our end user

Reply:
I can. I think it might be convention for it to be above the error boundary in case that is i18n'ed as this is how `infra` does it at the moment:

https://github.com/elastic/kibana/blob/master/x-pack/plugins/infra/public/apps/start_app.tsx#L34

Let me know if I should move it below, but right now it is consistent with other usages.

Reply:
Changed it to be above the EuiErrorBoundary. Not seeing issues with it, so we are ok.